server/routes/serviceProviderReferrals/interventionProgressView.ts

```diff
@@ -110,12 +110,24 @@ export default class InterventionProgressView {
           { text: `Session ${row.sessionNumber}` },
           { text: `${row.appointmentTime}` },
           { text: tagMacro(row.tagArgs as TagArgs) },
-          { html: `${row.linkHtml}` },
+          { html: this.linkHtml(row.links) },
         ]
       }),
     }
   }
 
+  private linkHtml(links: { text: string; href: string }[]): string {
+    const linkArray = links.map(link => {
+      if (link.text && link.href) {
```

Comment:
When would this condition be false?

Reply:
When link/href is an empty string (which we set it to if we don't want to display a link)

Reply:
Hmm, could we use `null` to indicate this instead? It's clearer, I think.

Reply:
(and it's what we do elsewhere)

Reply:
Sorry, I got this confused with the PP code path (where we do set them to empty strings), they're always set here. I've removed the check here and added a separate commit for making the PP ones nullable.